src/components/Saturation.tsx

```diff
@@ -1,19 +1,28 @@
 import React, { useCallback } from "react";
+
 import Interactive from "./Interactive";
-import formatClassName from "../utils/formatClassName";
+
 import styles from "../styles.css";
-import hsvToHslString from "../utils/hsvToHslString";
+import { AnyColor, HSV } from "../types";
+import { hsvToHslString } from "../utils/convert";
+import formatClassName from "../utils/formatClassName";
+
+interface Props {
+  hsv: HSV;
+  onChange: (newColor: AnyColor) => void;
+}
 
-const Saturation = ({ hsv, onChange }) => {
+const Saturation = ({ hsv, onChange }: Props) => {
   const handleMove = useCallback(
     (interaction) => {
       // Saturation and brightness always fit into [0, 100] range
       onChange({
+        h: hsv.h,
         s: interaction.left * 100,
         v: 100 - interaction.top * 100,
       });
     },
-    [onChange]
+    [hsv.h, onChange]
```

Comment:
Could we avoid adding `hsv.h` here somehow?

Reply:
For example just by a simple solution like this:
```
      onChange({
        s: interaction.left * 100,
        v: 100 - interaction.top * 100,
      }: { s: number, v: number });
```

Reply:
Sorry, absolutely.